bedrock/security/models.py

```diff
@@ -67,7 +67,7 @@ def save(self, force_insert=False, force_update=False,
 class SecurityAdvisory(models.Model):
     id = models.CharField(max_length=8, primary_key=True, db_index=True)
     title = models.CharField(max_length=200)
-    impact = models.CharField(max_length=100)
+    impact = models.CharField(max_length=100, null=True)
```

Comment:
Why not `default=''` instead of `null=True`. Making `CharFields` nullable creates a situation where this field will eval to False both when it's `null` and when it's empty string `''` which may result in unexpected results?